content/en/project/ADHD_frontoparietal_network/index.md

```diff
@@ -0,0 +1,87 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2025-06-15" # Date you first upload your project.
+# Title of your project (we like creative title)
+title: "Analysing Variability in Frontoparietal Activity in Children with and without ADHD"
+
+# Name of the collaborator
+name: [Hu Ding Xuan]
+
+# Your project GitHub repository URL
+github_repo: https://github.com/nauxgnid/hu_project
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [adhd, fmri, connectivity, differences]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "This study examines dorsolateral prefrontal cortex (dlPFC) and posterior parietal cortex (PPC) connectivity and dlPFC BOLD time series in ADHD versus typically developing (TD) children during the cued stop-signal task (CSST) using fMRI data from OpenNeuro ds005899. It is hypothesised that stronger dlPFC-PPC connectivity will be found in the ADHD group."
+
```

Comment:
Please add a cover image for your project using `image: "cover.png"`. The image needs to be in the same folder as `index.md`.